services-camel/src/main/webapp/WEB-INF/service-context.xml

```diff
@@ -55,10 +55,249 @@
 		<property name="serverUri" value="${fcrepo.serverUri}" />
 	</bean>
 
-	<bean id="pids" class="edu.unc.lib.dl.fcrepo4.PIDs">
-		<property name="repository" ref="repository" />
+	<bean id="pids"
+		class="org.springframework.beans.factory.config.MethodInvokingFactoryBean">
+		<property name="targetClass" value="edu.unc.lib.dl.fcrepo4.PIDs" />
+		<property name="targetMethod" value="setRepository" />
+		<property name="arguments">
+			<list>
+				<ref bean="repository" />
+			</list>
+		</property>
+	</bean>
+	
+	<!-- Solr settings -->
+	<bean id="collectionsPid" class="org.springframework.beans.factory.config.MethodInvokingFactoryBean">
+		<property name="targetObject">
+			<ref bean="tripleStoreQueryService" />
+		</property>
+		<property name="targetMethod">
+			<value>fetchByRepositoryPath</value>
+		</property>
+		<property name="arguments">
+			<list>
+				<value>/Collections</value>
+			</list>
+		</property>
+	</bean>
+
+	<bean id="solrFullUpdatePipeline"
+		class="edu.unc.lib.dl.data.ingest.solr.indexing.DocumentIndexingPipeline">
+		<property name="filters">
+			<list>
+				<ref bean="setDisplayOrderFilter" />
```

Comment:
Switch to a different Filter, this one isn't updated yet and is probably going to be removed.  A good alternative is setRecordDatesFilter, which I thought you'd done on the previous branch?